BUILDING.md

```diff
@@ -233,24 +233,24 @@ The Node.js project supports Python >= 3 for building and testing.
 
 #### Unix prerequisites
 
-* `gcc` and `g++` >= 8.3 or newer, or
+* `gcc` and `g++` >= 8.3 or newer
 * GNU Make 3.81 or newer
-* Python 3.6, 3.7, 3.8, 3.9, or 3.10 (see note above)
+* Python 3.6, 3.7, 3.8, 3.9, or 3.10 (see note above) with pip installed
```

Comment:
On clean install the `make coverage` fails if pip is not installed here: https://github.com/nodejs/node/blob/f8035ecbbd9f32ec5ae398495cd645f00c4b0c51/Makefile#L222

Reply:
Perhaps it would be better to instead include this as a note below mentioning that it's *only* required for code coverage reporting?

Reply:
I included it here because it makes it easier to just include pip in the package manager commands below as well, and it seemed like the only missing dependency to get the commands running. 

That being said, if it makes more sense to mention it separately in the section about coverage, then that's fine too

Reply:
Perhaps a sub-bullet?:

```suggestion
* Python 3.6, 3.7, 3.8, 3.9, or 3.10 (see note above)
  * For test coverage, your Python installation must include pip.
```

Reply:
That makes sense, updated